tests/early-data.rs

```diff
@@ -33,12 +33,32 @@ impl<T: AsyncRead + Unpin> Future for Read1<T> {
     }
 }
 
+pin_project! {
+    struct TlsStreamEarlyWrapper<IO> {
+        #[pin]
+        inner: server::TlsStream<IO>
+    }
+}
+
+impl<IO> AsyncRead for TlsStreamEarlyWrapper<IO>
+where
+    IO: AsyncRead + AsyncWrite + Unpin,
+{
+    fn poll_read(
+        self: Pin<&mut Self>,
+        cx: &mut Context<'_>,
+        buf: &mut ReadBuf<'_>,
+    ) -> Poll<io::Result<()>> {
+        return self.project().inner.poll_read_early_data(cx, buf);
+    }
+}
+
 async fn send(
     config: Arc<ClientConfig>,
     addr: SocketAddr,
     data: &[u8],
     vectored: bool,
-) -> io::Result<(TlsStream<TcpStream>, Vec<u8>)> {
+) -> io::Result<(client::TlsStream<TcpStream>, Vec<u8>)> {
```

Comment:
This change seems unrelated? If so, prefer to avoid it.

Reply:
I gave a more qualified name because I use server::TlsStream as well. should I still revert it?